streams/in.m3u

```diff
@@ -636,3 +640,7 @@ https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 https://satsangtv.sanskargroup.in/satsangtvlive.m3u8
 #EXTINF:-1 tvg-id="" status="error",Divya TV [Geo-blocked]
 https://sneh-z5api.herokuapp.com/?c=0-9-divyatv
+#EXTINF:-1 tvg-id="" status="error",Toonz Kids (Indonesian dub) (576p) [Geo-blocked]
+http://210.210.155.35/x6bnqe/s/s81/01.m3u8
+#EXTINF:-1 tvg-id="" status="error",Toonz Kids (Indonesian dub) (576p)
+http://210.210.155.37/x6bnqe/s/s81/01.m3u8
```

Comment:
```suggestion
http://210.210.155.37/x6bnqe/s/s81/index.m3u8
```

Reply:
> http://210.210.155.37/x6bnqe/s/s81/index.m3u8

Error 404

Reply:
my bad, http://210.210.155.37/x6bnqe/s/s81/index1.m3u8

Reply:
yup, in this case, add http://210.210.155.37/x6bnqe/s/s81/02.m3u8 as alt.